mne/decoding/search_light.py

```diff
@@ -0,0 +1,491 @@
+# Author: Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from .mixin import TransformerMixin
+from .base import BaseEstimator  # XXX why are these sklearn object copied in weird places. We should just have an externals.sklearn? # noqa
+from ..parallel import parallel_func
+
+
+class SearchLight(BaseEstimator, TransformerMixin):
+    """Search Light.
+
+    Fit, predict and score a series of models to each subset of the dataset
+    along the third dimension.
+
+    Parameters
+    ----------
+    base_estimator : object | None (default=None)
+        The base estimator to iteratively fit on a subset of the dataset. If
+        None, then the estimator is a Logistic Regression.
```

Comment:
I'm trying to keep it to its simplest. I'm even wondering whether I shouldn't change it have no default, so as to force the user to explicit the problem.